src/js/themes/base.d.ts

```diff
@@ -560,14 +560,44 @@ export interface ThemeType {
     pad?: PadType;
     size?: string;
     toggle?: {
+      container?: {
+        background?: BackgroundType;
+        border?: BorderType;
+        height?: string;
```

Comment:
i wanted to allow a non-extend way to have a container with a different height and width (as the [current toggle storybook example](https://github.com/grommet/grommet/blob/master/src/js/components/CheckBox/stories/CustomToggle.js#L52) sets a different height in the extend) - but also wanted to indicate that the size would also inform the height if height is not defined (since that is the current-day functionality). 

perhaps we could add a separate `width` string as well, or optionally make `size` and object with `width` and `height`? i'm not sure what the most semantic solution would be for this so open to anything